amlb/singularity.py

```diff
@@ -127,15 +127,15 @@ def _image_exists(self):
         try:
             # We pull from docker as there are not yet singularity org accounts
             run_cmd("singularity pull {output_file} docker://{image}".format(
-                image=self._container_image_name(return_docker_name=True),
+                image=self._container_image_name(as_docker_image=True),
                 output_file=self._image_name,
             ), _live_output_=True)
```

Comment:
@franchuterivera any idea why the live output doesn't seem to work with `singularity pull`?
My logic has been working with all commands I've been using until now, including when building or pulling docker image, but here I don't see any output/feedback when pulling singularity image, which is very annoying because it's hard to know if it's really doing anything or if it just got stuck...

Reply:
looks like singularity pull is buffering the output, idk..

Reply:
Yeah, same behavior on my end... there is an output, but after the command has completed, which takes like 5 mins... If I found a solution I would let you know! thanks for looking into this...